src/ResourceManager/EventHub/Commands.EventHub/help/Get-AzureRmEventHubKey.md

```diff
@@ -125,18 +153,16 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
-### CommonParameters
```

Comment:
I'm not sure why the common parameters are being removed, let me know if regenerating after a build does not fix this issue.

Reply:
Resolved